src/Pkg/PParser.hs

```diff
@@ -2,7 +2,13 @@
 #if !(MIN_VERSION_base(4,8,0))
 {-# LANGUAGE OverlappingInstances #-}
 #endif
-
+--  ------------------------------------------------------------- [ PParser.hs ]
+--  Module    : PParser.hs
+--  Copyright : (c) The Idris Community.
+--  License   : see LICENSE
+--
+--  The parser for Idris ipkg files.
```

Comment:
This is a good thing to add - but shouldn't it be a [Haddock module description](https://www.haskell.org/haddock/doc/html/ch03s03.html) instead of a comment?